### ISSUE TITLE
JPEG quality range in CIR dataset

### ISSUE THREAD

**Issue by sojinleeme** [renyulin-f/MoE-DiffIR#8]
Dear @renyulin-f,

Thank you for interest research!

I downloaded the CIR dataset and noticed that **the distortion levels of JPEG** differ between the training and some test sets:
- Training set: Q=10, 15, 20
- Test set (BSDS, Classic5, and ICB): Q=10, 15, 20
- Test set (DIV2K_test and LIVE1): **Q=5, 10, 15**

In Appendix B, Table 5 reports results for **JPEG Q=10, 15, 20**, and the unseen task dataset, particularly Cross Degree JPEG, includes Q=5.

Could you check the codec file in **CIR_dataset/Test_Compressed_Dataset/DIV2K_test and LIVE1**?
